src/main/java/seedu/workbook/model/internship/Role.java

```diff
@@ -51,6 +52,21 @@ public boolean equals(Object other) {
                         && value.equals(((Role) other).value)); // state check
     }
 
+    public String toUpperCase(String roleName) {
```

Comment:
technically violating DRY, up to you if you want to move this common method to one of the Util classes and call it in `Company.java` and here